src/phoenix/trace/llama_index/debug_callback.py

```diff
@@ -0,0 +1,50 @@
+import logging
+from typing import Any, Dict, List, Optional
+
+from llama_index.callbacks.base_handler import BaseCallbackHandler
+from llama_index.callbacks.schema import CBEventType
+
+logger = logging.getLogger(__name__)
+logger.addHandler(logging.NullHandler())
+
+CBEventID = str
+
+
+class LlamaIndexDebugHandler(BaseCallbackHandler):
```

Comment:
```suggestion
class DebugCallbackHandler(BaseCallbackHandler):
```
Probably sufficient since it's already scoped by the module.

Reply:
Yeah, that's much better, done!